docs/make.jl

```diff
@@ -2,16 +2,11 @@ using Documenter
 using Literate, JSON
 using COBREXA
 
-# Note: required to deploy the doc from Gitlab CI instead of Travis
-ENV["TRAVIS_REPO_SLUG"] = "LCSB-BioCore/COBREXA.jl"
-ENV["TRAVIS_BRANCH"] = "master"
-
-# set the merge/pull request ID
-if "CI_EXTERNAL_PULL_REQUEST_IID" in keys(ENV)
-    ENV["TRAVIS_PULL_REQUEST"] = ENV["CI_EXTERNAL_PULL_REQUEST_IID"]
-else
-    ENV["TRAVIS_PULL_REQUEST"] = "false"
-end
+# some settings
+dev_docs_folder = "dev"
+pages_branch = "gh-pages"
+github_repo_slug = "LCSB-BioCore/COBREXA.jl"
+delete!(ENV, "GITHUB_REPOSITORY")
```

Comment:
I think we should remove this env variable on Gitlab and not here - can you change?

Reply:
I can't modify gitlab settings but you can safely erase this one there now. Then I can remove this from here

Reply:
we can remove it, but then let's use other variables (see comment below)

Reply:
Removed completely (see #500)